custom/icds_reports/reports/awc_reports.py

```diff
@@ -335,7 +335,7 @@ def get_awc_reports_pse(config, month, domain, show_test=False):
 def get_awc_reports_maternal_child(domain, config, month, prev_month, show_test=False, icds_feature_flag=False):
 
     def get_data_for(date):
-        age_filters = {'age_tranche': 72} if icds_feature_flag else {'age_tranche__in': [0, 6, 72]}
```

Comment:
Hey, Did you check below how this age_filters is being used?. 
I think its being used like `select all from table excluding age_tranche=60` that would include all children accept children of age 5 years.

Reply:
Since we need to include children less than 72 month of age. You could reuse the `include_record_by_age_column` function created in this PR https://github.com/dimagi/commcare-hq/pull/26125/files and work this out similarly.